crates/typst/translations/en.txt

```diff
@@ -4,4 +4,5 @@ equation = Equation
 bibliography = Bibliography
 heading = Section
 outline = Contents
-raw = Listing
\ No newline at end of file
+raw = Listing
+page = Page
```

Comment:
I agree that it's not nice that the supplement will be in uppercase in English by default. "As seen on Page 10" feels wrong compared to "As seen on page 10".

Reply:
I'll change it to lowercase then. I guess the cases where it is at the start of the sentence are much rarer anyways. Implementing logic to change the case is out of the scope of this PR anyways.